src/net/chat.rs

```diff
@@ -1811,6 +1811,13 @@ impl ConversationP2P {
                         break;
                     }
                 }
+                Err(net_error::PermanentlyDrained) => {
```

Comment:
I'd be curious to know what kinds of shenanigans you encountered that prompted you to debug connection closures :thinking:

Reply:
Just trying to reduce some excessive log verbosity -- this PermanentlyDrained error was getting logged as `info` level, while not really conveying any useful information.

Reply:
Ah.